recipe/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "2.12.0" %}
+{% set name = "tensorflow-text" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tensorflow/text
+  git_rev: v{{ version }}
+  patches:
+    - 0001-fix-build.sh
+
+build:
+  number: 0
+  # skipping s390x and win-64 since we don't have tf for it
+  skip: True  #[s390x or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - git  # [not win]
+    - patch  # [not win]
+  host:
+    # conda build requirements
+    - tensorflow {{ version }}
+  run:
+    - {{ pin_compatible("tensorflow", min_pin="x.x.x", max_pin="x.x") }}
+
+test:
+  imports:
+    - tensorflow
+  commands:
+    - conda inspect linkages -p $PREFIX tensorflow-text  # [linux]
```

Comment:
DO you think we could try to run an example from https://github.com/tensorflow/text/blob/v2.12.0/examples/keras_example_174.ipynb? There is also tutorials in https://github.com/tensorflow/text/tree/f0f675cb1e24cfd0e42aa95f3136b7aee3c91bc6/docs/tutorials that could we also look into for tests to run.

Reply:
I'm not really sure how to use these examples without having a result evaluation or a comparison term.
The only way I would consider using these tests is as a mere smoke test to see if no exceptions or fatal errors arise.

Reply:
That's the idea yes. Smoke tests are fine.